### ISSUE TITLE
need to press ":" twice in order to see it appear

### ISSUE THREAD

**Issue by dsc8x** [atom/atom-keymap#39]
When i press ":" the first time nothing happens. But when I type it one more time the ":'" appears.

Note: I do not use a default US layout. To Type characters like {}():;<>[] I use the capslock key plus a normal letter. All those mappings work fine with atom, except of the ":" which is mapped to Capslok + g  (g = key code 0x29)

**Comment by penafo** [comment]
The colon ( : ) is also not working on my keyboard either.

How come this bug is from 30 Jun 2014 and wasn't closed yet? After 1 hour using Atom I thought it was going to be this time I was leaving Brackets :| Such a small bug and so big the annoyances caused. Makes it impossible to write CSS...

**Comment by 50Wliu** [comment]
@penafo which keyboard layout are you using?

**Comment by epegzz** [comment]
Sad to hear it's still not fixed. Was about to give Atom another try soon.

**Comment by 50Wliu** [comment]
@epegzz While I'm not sure #35 directly pertains to you, it might be worth it to subscribe there as the fix for #35 will probably fix your issue as well.

**Comment by epegzz** [comment]
Yeah, just saw that myself, sounds promising. Any idea when the current electron branch will be used in another Atom release?

**Comment by 50Wliu** [comment]
There's currently no ETA.  Right now we're trying to update to the latest 0.36.x version, and then we'll probably start updating to 0.37.x.

**Comment by penafo** [comment]
@50Wliu: Portuguese layout keyboard --> [wikipedia image](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2c/KB_Portuguese.svg/2000px-KB_Portuguese.svg.png)

I already commented on [#35](https://github.com/atom/atom-keymap/issues/35#issuecomment-203156527) the problem that was originating it. 

Atom shouldn't allow external plugins to change users keyboard's "normal" keys in that manner.... allowing a plugin to change the function of a key as basic as the colon is going to kill the user-experience on Atom as a whole.

Why not forcing all 3rd party plugins to use ONLY something like CTRL+SHIFT+somthing for theirs shortcuts and prompting the user to change them in case on install a new plugin some shortcuts "collide" with already registered shortcuts?
